File: packages/android/src/main/kotlin/gov/civui/components/CivBulkActions.kt
// CivUI — CivBulkActions for Jetpack Compose
// "N items selected" bar with action affordances. Mirrors civ-bulk-actions.
//
// Placeholder body — see .claude/rules/audit-debt.md → "Native platform implementation pass".

package gov.civui.components

import androidx.compose.foundation.layout.Row
import androidx.compose.runtime.Composable
import androidx.compose.ui.Modifier

@Composable
fun CivBulkActions(
    count: Int = 0,
    itemName: String = "",
    itemNamePlural: String = "",
    clearLabel: String = "",
    hideClear: Boolean = false,
    modifier: Modifier = Modifier,
    content: @Composable () -> Unit = {},
) {
    Row(modifier = modifier) {
        // Placeholder — see audit-debt.md for native implementation status.
        content()
    }
}

File: packages/ios/Sources/CivUI/CivBulkActions.swift
// CivUI — CivBulkActions for SwiftUI
// "N items selected" bar with action affordances. Mirrors civ-bulk-actions.
//
// Placeholder body — the prop surface satisfies schema parity. See
// .claude/rules/audit-debt.md → "Native platform implementation pass".

import SwiftUI

public struct CivBulkActions: View {
    /// Number of currently-selected items.
    public var count: Int

    /// Singular item label (e.g. `row`).
    public var itemName: String

    /// Plural item label (e.g. `rows`).
    public var itemNamePlural: String

    /// Label for the clear-selection button.
    public var clearLabel: String

    /// Hide the clear-selection affordance entirely.
    public var hideClear: Bool

    public init(
        count: Int = 0,
        itemName: String = "",
        itemNamePlural: String = "",
        clearLabel: String = "",
        hideClear: Bool = false
    ) {
        self.count = count
        self.itemName = itemName
        self.itemNamePlural = itemNamePlural
        self.clearLabel = clearLabel
        self.hideClear = hideClear
    }

    public var body: some View {
        EmptyView()
    }
}
